cli/README.md

```diff
@@ -122,3 +122,7 @@ Options:
   -v, --verbose               Enable verbose logging. (default: false)
   -h, --help                  display help for command
 ```
+
+### Releasing the CLI
+
+To deploy an updated version of the CLI to NPM run the [Publish to NPM](https://github.com/rocketstack-matt/architecture-as-code/actions/workflows/publish-cli-to-npm.yml) Action.
```

Comment:
is manually running the action definitely the way we want this flow to work? It's currently setup to trigger when you do a GH release which I think is nicer since there is a clear history on the GH page itself (at least it should be, I recall you mentioning that you didn't think it was working, but the config is there)
![image](https://github.com/user-attachments/assets/8fcb1cee-e832-4a37-98df-a4f73218a7e1)

Reply:
Hmmm - @rocketstack-matt can I pick up this issue off of you. I have a feeling the steps will of changed with moving to npm workspaces. I thought I had covered most of the use-cases but this appears to be another.